data/Caldwell_2016/metadata.yml

```diff
@@ -23,7 +23,7 @@ contributors:
     additional_role: contact
   - last_name: Sanson
     given_name: Gordon
-    ORCID: .na.character
+    ORCID: 0000-0002-7207-2754
```

Comment:
probably remove this. I think he is retired and looking on GoogleScholar there are other Gordon Sanson's